pom.xml

```diff
@@ -126,6 +126,7 @@
         <lucene.version>8.11.2</lucene.version>
         <bitmap.version>1.0.1</bitmap.version>
         <prometheus.version>0.16.0</prometheus.version>
+        <flink.version>1.18.1</flink.version>
     </properties>
```

Comment:
If we introduce flink dependencies here, will it affect the flink version in the **mixed-format** and other dependencies? **flink-optimizer.flink-version** can specify flink dependencies

Reply:
No, the dependency version of Flink is specified separately in other modules.